### DISCUSSION TITLE
Questions about Animation (components) parameters

### DISCUSSION THREAD

**Discussion by apollinariya06** [BrokenSource/DepthFlow#31] [Q&A]
Hello, 
I've installed the project `from Source` and read the documentation and help, but I'm unable to achieve the result I'd like. 
I've been trying to use this command for example :  `depthflow linear --target zoom -t0 0.0 -t1 1.0 -y0 2.0 -y1 1.0 input -i "url path" main` for a zoom out effet (I assume from my unserstanding of the instructions) but this doesn't update the render. It looks like it's rendering with the default, same with other tests I did with sine and constant.
Can you guide me ? Thank you very much

**Comment by Tremeschin** [comment]
Ah, sorry, uncomment the lines 132-135 in [this file](https://github.com/BrokenSource/DepthFlow/blob/1f1edb4e9db0c297ae05425fac025cea5f5f52e5/DepthFlow/Scene.py#L132), or write a custom animation in its place

Animation system is a bit incomplete at the moment and disabled by default, I was working on it but got some IRL stuff to do (+internship), and didn't rewrite the default preset to it. I'll be busy for a while and doing mostly light work on the code in the next month or so

**Comment by apollinariya06** [comment]
Thank you for your quick answer, I'll try that !
